RELEASE.txt

```diff
@@ -41,11 +41,22 @@ make a PR to update these notes after you are done with the release. ;-)
   - Discuss with others, and make the changes directly to v<major>.<minor>.x branch.
   - Once the consensus is found, ask the project maintainers to merge the PR.
 
-- In the master branch, update the version number in ``skimage/__init__.py``
-  to the next ``-dev`` version, commit, and push.
+  - In the master branch, create a new tag with an updated version number and
+    append ``.dev`` to it. python-versioneer will take care of updating the
+    version number in the source file.
```

Comment:
What exactly is happening here? I'm specifically confused about why there is no version number anywhere in the changes... Is the version number always determined "live"? What about when we distribute the source without the git history?

Reply:
sdist seems ok, see issue for more [details](https://github.com/scikit-image/scikit-image/issues/3144#issuecomment-394112573).

Review comments seem to get hidden eventually and this is a real concern.

Reply:
@hmaarrfk yes it is, there are quite a few rants about GitHub's review process floating around. But it's simple and very useful in the simple cases so I cut them a bit of slack.